npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Our existing shrinkwrap config had enzyme locked to 2.2.0. The `containsMatchingElement` and related methods are only available in 2.3.0, so I ticked up that dep and rebuilt shrinkwrap. Looks like it hadn't been run in a very long time as there were lots of changes.